Resources/doc/commands/generate_bundle.rst

```diff
@@ -32,17 +32,26 @@ forget to pass all needed options:
 
     If the ``generate:bundle`` command returns an error about registering the
     bundle namespace in ``composer.json``, add the following line to your
-    ``composer.json`` file within the ``psr-4`` section
-    ``"Acme\\Bundle\\BlogBundle\\": "src/Acme/Bundle/BlogBundle"``:
+    ``composer.json`` file within the ``psr-4`` section:
 
-.. code-block:: json
+    ``"Acme\\Bundle\\BlogBundle\\": "src/Acme/Bundle/BlogBundle"``
 
-    "autoload": {
-        "psr-4": {
-            "Acme\\Bundle\\BlogBundle\\": "src/Acme/Bundle/BlogBundle"
+    (for example:
+
+    .. code-block:: json
+
+        "autoload": {
+            "psr-4": {
+                "Acme\\Bundle\\BlogBundle\\": "src/Acme/Bundle/BlogBundle"
+            },
+            "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
         },
-        "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
-    },
+
+    ), then execute the following command to regenerate the autoload files:
+
+    .. code-block:: bash
+
+        $ composer dump-autoload
```

Comment:
@fabpot I don't know if it should, but it seems needed even in dev (that or a `composer install`, which includes autoloader dump), see for example https://pastebin.com/CF0DEQbt and https://pastebin.com/hy04dQpX
(by the way, in prod we would probably add `-o`)